packages/themes/src/presets/bluePlanet.css

```diff
@@ -144,7 +137,7 @@
   color: var(--default-theme-background-1) !important;
 }
 .active_page .sidebar-heading-type {
-  color: var(--sidebar-color-1) !important;
+  color: var(--default-sidebar-color-1) !important;
```

Comment:
We need to use the unprefixed var as the fallback, don’t we?

```css
 color: var(--default-sidebar-color-1, var(--sidebar-color-1)) !important;
```

Reply:
Good catch, this is leftover theme code from closed source repo so I can actually just delete that :)